tests_e2e/orchestrator/lib/agent_test_loader.py

```diff
@@ -33,8 +33,8 @@ class TestSuiteInfo(object):
     name: str
     # The tests that comprise the suite
     tests: List[Type[AgentTest]]
-    # An image or image set (as defined in images.yml) specifying the images the suite must run on.
-    images: str
+    # Images or image sets (as defined in images.yml) on which the suite must run.
+    images: List[str]
```

Comment:
I added support for multiple images/image sets so now this is a List, you'll see several changes wrapping references to this member in for loops